backend/pkg/modules/internal/bfla/bfladetector/learn_and_detect_bfla.go

```diff
@@ -57,10 +59,12 @@ func NewBFLADetector(ctx context.Context, apiInfoProvider apiInfoProvider, event
 		apiInfoProvider:          apiInfoProvider,
 		authzModelsMap:           recovery.NewPersistedMap(sp, AuthzModelAnnotationName, reflect.TypeOf(AuthorizationModel{})),
 		tracesCounterMap:         recovery.NewPersistedMap(sp, AuthzProcessedTracesAnnotationName, reflect.TypeOf(1)),
+		findingsMap:              recovery.NewPersistedMap(sp, BFLAFindingsAnnotationName, reflect.TypeOf(1)),
```

Comment:
Make sure to delete them when a reset is received.

Reply:
good point, fixed